src/link.js

```diff
@@ -12,22 +12,26 @@ const linkDialogImpl = updateAttrsDialog(
       type: "string",
       title: gettext("Title"),
     },
-    openInNewWindow: {
+  }
+
+  if (!options.hideOpenInNewWindow)
+    properties.openInNewWindow = {
       type: "boolean",
       title: gettext("Open in new window"),
-    },
-  },
-  {
+    }
+
+  return updateAttrsDialog(properties, {
     title: gettext("Edit Link"),
-  },
-)
-const linkDialog = async (editor, attrs) => {
+  })(editor, attrs)
+}
+const linkDialog = async (editor, attrs, options) => {
   attrs = attrs || {}
   attrs.openInNewWindow = attrs.target === "_blank"
-  attrs = await linkDialogImpl(editor, attrs)
+  attrs = await linkDialogImpl(editor, attrs, options)
   if (attrs) {
     if (attrs.openInNewWindow) {
       attrs.target = "_blank"
+      attrs.rel = "noopener noreferrer nofollow"
```

Comment:
Since we're already looking at this, I'd be good with removing some of the options here. `noopener` is important and we certainly have to keep that, but I'm less sure about `noreferrer` and especially not about `nofollow` when it's used in a CMS. It's different for comment boxes or similar use cases of course, but then we would want server-side filtering anyway.